packages/constants/src/payload/payload.types.ts

```diff
@@ -52,6 +52,10 @@ export interface SignMessageRequestPayload {
   message: string;
 }
 
+export interface NftImageRequestPayload {
```

Comment:
This is not shared with the API somehow, I don't think it should live here,  you should keep it within the component.

Reply:
Moved